healdata.org/manifests/hatchery/hatchery.json

```diff
@@ -2,7 +2,12 @@
   "user-namespace": "jupyter-pods",
   "sub-dir": "/lw-workspace",
   "user-volume-size": "10Gi",
-  "pay-models": [{
+  "default-pay-model": {
```

Comment:
I would wait a bit for this change since the hatchery in prod doesn't support this now